mathesar/exception_handlers.py

```diff
@@ -17,4 +31,24 @@ def mathesar_exception_handler(exc, context):
         else:
             raise exc
 
+    if response is not None:
+        # Check if conforms to the api spec
+        if is_pretty(response.data):
+            # Validation exception converts error_codes from integer to string, we need to convert it back into
```

Comment:
I'm not sure what this means.  Consider a more descriptive function name instead of `fix_error_response`.